src/BulletCollision/Gimpact/btBoxCollision.h

```diff
@@ -234,7 +234,24 @@ ATTRIBUTE_ALIGNED16	(class) btAABB
 
 	btAABB()
 	{}
-
+	
```

Comment:
What operator implementations was it using before for those? I'm not too familiar with the GIMPACT internals (never use it).